fl4health/utils/dataset.py

```diff
@@ -81,3 +81,13 @@ def __getitem__(self, index: int) -> Tuple[Dict[str, torch.Tensor], torch.Tensor
     def __len__(self) -> int:
         first_key = list(self.data.keys())[0]
         return len(self.data[first_key])
+
+
```

Comment:
Adding this in to be used for the tests, but I think it might also be useful more generally. So I put it here in this part of the library.